tests/integration/istio/istio_test.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2016-2018, Pulumi Corporation.  All rights reserved.
+
+package ints
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/pulumi/pulumi/pkg/testing/integration"
+)
+
+func TestIstio(t *testing.T) {
+	kubectx := os.Getenv("KUBERNETES_CONTEXT")
+
+	if kubectx == "" {
+		t.Skipf("Skipping test due to missing KUBERNETES_CONTEXT variable")
+	}
+
+	integration.ProgramTest(t, &integration.ProgramTestOptions{
+		Dir:          "step1",
+		Dependencies: []string{"@pulumi/kubernetes"},
+		Quick:        true,
+		SkipRefresh:  true,
+		ExtraRuntimeValidation: func(t *testing.T, stackInfo integration.RuntimeValidationStackInfo) {
+			// assert.NotNil(t, stackInfo.Deployment)
+			// assert.Equal(t, 143, len(stackInfo.Deployment.Resources))
+
+			frontend := stackInfo.Outputs["frontendIp"].(string)
+			for range []int{1, 2, 3, 4, 5, 6, 7, 8, 9, 10} {
```

Comment:
Any reason not to use a simple for loop?

```suggestion
			for i := 1; i <= 10; i++ {
```